internal/controllers/clusterextension_controller_test.go

```diff
@@ -148,16 +148,14 @@ func TestClusterExtensionResolutionSuccessfulUnpackFails(t *testing.T) {
 			Image:   "quay.io/operatorhubio/prometheus@fake1.0.0",
 		}, &v, nil, nil
 	})
-	res, err := reconciler.Reconcile(ctx, ctrl.Request{NamespacedName: extKey})
+	res, _ := reconciler.Reconcile(ctx, ctrl.Request{NamespacedName: extKey})
 	require.Equal(t, ctrl.Result{}, res)
-	require.Error(t, err)
 
 	t.Log("By fetching updated cluster extension after reconcile")
 	require.NoError(t, cl.Get(ctx, extKey, clusterExtension))
 
 	t.Log("By checking the status fields")
-	expectedBundleMetadata := ocv1alpha1.BundleMetadata{Name: "prometheus.v1.0.0", Version: "1.0.0"}
-	require.Equal(t, expectedBundleMetadata, clusterExtension.Status.Resolution.Bundle)
+	require.Equal(t, ocv1alpha1.BundleMetadata{Name: "prometheus.v1.0.0", Version: "1.0.0"}, clusterExtension.Status.Resolution.Bundle)
```

Comment:
I would do the inverse here (keep what is being removed, get rid of what is being added)

Reply:
fixed 👍